### PR TITLE
Spinning up fresh Docker containers script

### PR DESCRIPTION
**Pull request by xnekv03**
I made this scripts for my myself and found it useful especially when I work on multiple PCs
Please consider merging to the main branch

Thank you

### CODE REVIEW

**Comment by eleftrik** [comment]
Hi @xnekv03 and thanks for your PR.
Just two small reviews. Could you please
- fix typos
- move your script inside `.docker/scripts/` folder
?

Thanks!

**Comment by xnekv03** [comment]
Hi @eleftrik,
sorry for typos. Fixed

**Comment by xnekv03** [comment]
Adding intl extension to Dockerfile

File: README.md [hunk 1]
```diff
@@ -421,12 +421,12 @@ MailHog web interface is available at
...
 
-## Spinning up fresh Docker containres
+## Spinning up fresh Docker containers
 
-You can remove all old Docker containers and spin up your entire project from scratch. Ideal if you're working on multiple environments.
+You can remove all old Docker containers and spin up your entire project from scratch. This is ideal if you're working in multiple environments.
 
 ```bash
-sh rundocker.sh
+sh .\.docker\scripts\rundocker.sh
```
Comment (eleftrik) [inline README.md]: @xnekv03 I think `/` should be used.

**Review by eleftrik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xnekv03) [reply README.md]: changed, however works both ways

**Comment by eleftrik** [comment]
@xnekv03 Thanks!

**Pull request merged**